docker-compose.yml

```diff
@@ -53,7 +37,7 @@ services:
     depends_on:
       - mongo
     command:
-      ["bash", "-c", "(bundle check || bundle install) && bundle exec rake up"]
+        ["bash", "-c", "(bundle check || bundle install) && bundle exec rake mongo:prepare && bundle exec rake up"]
```

Comment:
If we do the `bundle exec rake mongo:prepare` here, existing data will be always erased on startup.